script.js

```diff
@@ -1,5 +1,12 @@
 function getContacts(str) {
-  // your code here
+  var split = str.split(',')
+  var splitted = split.map(function (x){
+    return x.split(',')
```

Comment:
```suggestion
    return x.split(':')
```
ini di split by `":"` udah selesai pekerjaannya, bisa langsung `return splitted`